common/static/coffee/spec/discussion/view/discussion_view_spec_helper.coffee

```diff
@@ -14,6 +14,12 @@ class @DiscussionViewSpecHelper
           body: "",
           title: "dummy title",
           created_at: "2014-08-18T01:02:03Z"
+          ability: {
+              can_delete: false,
+              can_reply: true,
+              can_vote: false,
+              editable: false,
+            }
```

Comment:
@andy-armstrong, this is part of why the bug wasn't re-appearing for you. A necessary condition of the bug appearing is that the main post is *not* editable or deletable by the "current" user. With this change, any test-created threads are given these attributes (relative to the current user).